classes/output/renderer.php

```diff
@@ -471,10 +471,8 @@ protected function topcoll_section($section, $course, $onsectionpage, $sectionre
             $this->toggle_icon_set($sectioncontext);
             $sectioncontext['toggleiconsize'] = $this->tctoggleiconsize;
 
-            if ((!($section->toggle === null)) && ($section->toggle == true)) {
-                $sectioncontext['toggleopen'] = true;
-            } else {
-                $sectioncontext['toggleopen'] = false;
+            if (!is_null($toggle)) {
```

Comment:
Missing setting of $sectioncontext['toggleopen'] = false; if '$toggle' is null.

Reply:
Yes, I read the logic wrong in the first place, fixed!